harper-core/dictionary.dict

```diff
@@ -24101,7 +24101,7 @@ farewell/~NJVSg
 farina/~Ng
 farinaceous/J
 farm/~NVgd>ZGSz
-farmer/~Ng
+farmer/~NgS
 farmhand/NSg
```

Comment:
`farmer/S` generates the same `farmers` as `farm/Z` above. Though I'm not confident it gets all the right properties. I never use the flags like `/Z` in new stuff.